### DISCUSSION TITLE
Are these wallpapers able to steal passwords, info, etc?

### DISCUSSION THREAD

**Discussion by Shinare73** [rocksdanister/lively#1565] [Q&A] [answered]
In this world of heightened awareness of password and identity theft, I'm concerned if I run someone's wallpaper they can steal my passwords and info.  Is this possible with lively wallpapers?

**Comment by rocksdanister** [comment, marked as answer]
Lively supports different types of wallpapers files (right-click wallpaper -> About):
Video, GIF, Webpages and Application wallpaper (.exe)

Video and GIF types don't receive any input, so no.

Webpage by default is set to only receive mouse input (Settings -> Wallpaper -> Wallpaper input), so no; just to be safe don't load sketchy webpages as wallpaper (browser vulnerabilities attack) and don't enable keyboard input.

Application wallpaper can write their own input hooks so the mouse input setting in Lively is not enforced, don't load any unsafe application as wallpaper.
